vhdbuilder/packer/vhd-image-builder-arm64-gen2.json

```diff
@@ -53,10 +53,12 @@
         "SkipLinuxAzSecPack": "true",
         "os": "Linux",
         "now": "{{user `create_time`}}",
-        "createdBy": "aks-vhd-pipeline"
+        "createdBy": "aks-vhd-pipeline",
+        "image_sku": "{{user `img_sku`}}"
```

Comment:
do we need to set `img_sku` in `vhdbuilder/packer/settings.json`?

Reply:
For the purposes I made this commit for (faster / smoother debugging and monitoring experience during builds), I do not believe so. 

The IMG_SKU variable is available in the environment at the time of the resource group deployment object creation, so I just tagged the deployment group with it. That way it can be found easier if you're wondering what is going on with one of your builds. 

That being said, we do set the img sku for Windows builds it seems, but not Linux. Linux only gets the sig_image_name, which is just the base image. Would it be useful in the node provisioning process? I can add it in this or a separate PR if you would like. @cameronmeissner

Reply:
I just wanted to make sure this value is getting piped into the template correctly, though im assuming you've tested and it seems like it's good enough that it's in the environment - I can never remember if "user" variable space also takes from env, which it seems is the case